api/app/Http/Transformers/AuthTokenTransformer.php

```diff
@@ -19,26 +18,13 @@ class AuthTokenTransformer extends EloquentModelTransformer
     /**
      * Transform the token string into an response array.
      *
-     * @param  string  $object
+     * @param  string  $token
      * @return array
      */
-    public function transformItem($object)
+    public function transformItem($token)
     {
-        if (is_string($object)) {
-            return $this->transformToken($object);
-        }
-
-        return parent::transformItem($this->transformToken($object['token']));
-    }
-
-    protected function transformToken($token)
-    {
-        $token = new Token($token);
-        $jwtAuth = App::make('Tymon\JWTAuth\JWTAuth');
-
         return [
-            'token' => (string) $token,
-            'decodedTokenBody' => $jwtAuth->decode($token)->toArray(),
+            'token' => (string) $token
```

Comment:
The decodedTokenBody is not required by the app, but frequently used by developers when inspecting requests as it is easier to see it in plaintext rather than having to decode.